### DISCUSSION TITLE
Python Framwork django

### DISCUSSION THREAD

**Discussion by yattdevellopper** [community/community#134333] [GitHub Education]
### Hi everyone,

J'ai besoin d'aide pour les erreurs de syntaxe dans mes codes Merci !!!

**Comment by MuhamedHabib** [comment]
accept my response and send a messge to my email : khattat.mouhamedhabib@gmail.com ... i will debug your code!

**Comment by abhishek-sanwal** [comment]
Please share your code and the errors you are facing with what should be the outcome according to you.
